src/server.h

```diff
@@ -970,8 +970,12 @@ typedef struct redisDb {
     int id;                     /* Database ID */
     long long avg_ttl;          /* Average TTL, just for stats */
     unsigned long expires_cursor; /* Cursor of the active expire cycle. */
+    int resize_cursor;          /* Cron job uses this cursor to gradually resize dictionaries. */
     list *defrag_later;         /* List of key names to attempt to defrag one by one, gradually. */
-    clusterSlotToKeyMapping *slots_to_keys; /* Array of slots to keys. Only used in cluster mode (db 0). */
+    list *rehashing;            /* List of dictionaries in this DB that are currently rehashing. */
+    int dict_count;             /* Indicates total number of dictionaires owned by this DB, 1 dict per slot in cluster mode. */
+    unsigned long long key_count; /* Total number of keys in this DB. */
+    unsigned long long *binary_index;  /* Binary indexed tree (BIT) that describes cumulative key frequencies up until given slot. */
```

Comment:
```suggestion
    unsigned long long *slot_size_index_tree;  /* Binary indexed tree (BIT) that describes cumulative key frequencies up until given slot. */
```
Too verbose? I know we talked about this, but the name doesn't actually indicate what it is.

Reply:
Let's call it `slot_size_index`, and if anyone is interested in what kind of index, then they can go and read the comment.